Logger/FileLogger.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger: BaseLogger
+    {
+        private string fileName;
```

Comment:
Fields should be underscore PascalCase
```suggestion
        private string _FileName;
```